### DISCUSSION TITLE
3 Level design variable along with mixed variable definition.

### DISCUSSION THREAD

**Discussion by Atulsingh92** [anyoptimization/pymoo#220] [Q&A]
Hi, 

Is there a scope of definings design variables in levels? If so, how can I do so in a mixed variable definition?
(So for instance my values for a variable (density) needs to be either 40, 125 or 384, while the other variables are definied as usual, with bounds of xl and xu)

**Comment by blankjul** [comment]
For mixed variables check out https://pymoo.org/customization/mixed.html or if you need even more customization https://pymoo.org/customization/subset.html

Julian
